fpr/templates/format/form.html

```diff
@@ -1,5 +1,6 @@
 {% extends "fpr/app_layout.html" %}
 {% load url from future %}
+{% load bootstrap_tags %}
```

Comment:
django-forms-bootstrap, which is provided by the hosting app (Archivematica or the fpr-server).